base/operators.jl

```diff
@@ -5,8 +5,26 @@
 """
     <:(T1, T2)
 
-Subtype operator: returns `true` if and only if all values of type `T1` are
-also of type `T2`.
+Subtyping relation, defined between two types. The returned value is of type
+`Bool`.
+
+A type `S` is said to be a *subtype* of a type `T` if and only if we have `S <: T`.
```

Comment:
This is a tautology. I am not sure it helps to define `<:` in terms of itself.

Reply:
The idea was more to define what "subtype" means in Julia in terms of `<:`. Not sure if it's better now.